### ISSUE TITLE
examples: incorrect property in snippet example

### ISSUE THREAD

**Issue by jonschlinkert** [enquirer/enquirer#111]
See: https://github.com/enquirer/enquirer/issues/110

**Comment by zentek-marco-bruening** [comment]
This may be the wrong place to ask, but is there a doc about Snippet? Is there for example a way to change the color?

**Comment by jonschlinkert** [comment]
> Is there for example a way to change the color?

Yes, you should be able to pass a custom style on the options. I believe the style was named `typing`, to avoid conflict with other styles and give you complete control over the styling. 

```js
const colors = require('ansi-colors');
const question = {
  name: 'username',
  name: 'info',
  message: 'Fill out the following information:',
  styles: {
    typing: colors.red
  },
  template:  ''
};
```

Try also modifying the following styles, the same way as in the example:

- `heading`
- `primary`

Fwiw, this is going to be greatly improved in v3.0. I have some pretty great improvements planned for the next version!

**Comment by zentek-marco-bruening** [comment]
Thank you, *typing* did work, but *heading* and *primary* not

**Comment by jonschlinkert** [comment]
They work. But only when the values use those methods.
